src/main/java/bbc/forge/dsp/jaxrs/Caching500ErrorsForARevalidationPreventer.java

```diff
@@ -3,11 +3,13 @@
 import java.util.List;
 import java.util.Map;
 
-import javax.ws.rs.core.Response;
+import javax.ws.rs.container.ContainerRequestContext;
```

Comment:
Double set of imports? The Analyze->Inspect Code option in IntelliJ is great for spotting these when they slip in.